docs/modelserving/v1beta1/transformer/torchserve_image_transformer/README.md

```diff
@@ -42,24 +42,24 @@ def image_transform(instance):
 
 # for REST predictor the preprocess handler converts to input dict to the v1 REST protocol dict
 class ImageTransformer(kserve.Model):
-    def __init__(self, name: str, predictor_host: str):
+    def __init__(self, name: str, predictor_host: str, headers: Dict[str, str] = None):
```

Comment:
does init also need self.ready? @yuzisun 
looking at this for reference - it isn't exactly the same code though... we should probably test this still works soon https://github.com/kserve/kserve/blob/master/python/custom_transformer/model.py

Reply:
ye we need `self.ready`, the code snippet is just for demonstrating purpose to show how to override preprocess/postprocess in transformer for REST/gRPC case, the reference points to the actual working example.

Reply:
ok makes sense, I can add it to avoid confusion?